README.md

```diff
@@ -10,12 +10,20 @@ The Python SDK is under development. There are no compatibility guarantees nor p
 
 Prerequisites:
 
-* Python >= 3.7
+* [Python](https://www.python.org/downloads/) >= 3.7
 * [Poetry](https://python-poetry.org)
-* [Local Temporal server running](https://docs.temporal.io/clusters/quick-install/)
+* [Local Temporal server running](https://github.com/temporalio/temporal)
```

Comment:
Why change the link? Shouldn't we ask the do the quick install approach? And if that's not good enough for local quick install, can we change those docs?